src/components/DatePicker/index.jsx

```diff
@@ -167,12 +187,13 @@ export default class DatePickerComponent extends React.Component {
 
         <div className={styles.innerContainer}>
           <DatePicker
+            {...this.props}
```

Comment:
There are many we aren't covering here, such as `showTimeSelect`, `minDate`, `maxDate`... Some of these are being used on components in app-admin that were not migrated to nukleus yet. Instead of adding each one to our `propTypes`, doesn't it makes sense to import all props here?